docs/ci-integration/guides/gitlab-integration.md

```diff
@@ -0,0 +1,29 @@
+
+# GitLab CI/CD integration
+
+This example uses [GitLab CI/CD](https://docs.gitlab.com/ee/ci/) to build the Earthly target `+build`.
+
+
+```yml
+# .gitlab-ci.yml
+
+image: docker
+services:
+  - docker:dind
+
+before_script:
+    - apk update && apk add git
+    - wget https://github.com/earthly/earthly/releases/download/v0.6.1/earthly-linux-amd64 -O /usr/local/bin/earthly
+    - chmod +x /usr/local/bin/earthly
+    - /usr/local/bin/earthly bootstrap
+    - docker login -u $CI_REGISTRY_USER -p $CI_REGISTRY_PASSWORD $CI_REGISTRY
+
+earthly:
+  stage: build
+  script:
+    - earthly --ci --push -P +build

```

Comment:
Seems that the output is not colorized. Perhaps it's worth adding `FORCE_COLOR=1` in the env.